ports/atmel-samd/boards/ndgarage_ndbit6_v2/mpconfigboard.mk

```diff
@@ -0,0 +1,14 @@
+LD_FILE = boards/samd21x18-bootloader.ld
+USB_VID = 0x239A
+USB_PID = 0x8066
```

Comment:
Please use a different PID for new boards. (This is the same as the original: https://github.com/adafruit/circuitpython/blob/main/ports/atmel-samd/boards/ndgarage_ndbit6/mpconfigboard.mk#L3 )